dracut/30ignition/ignition-generator

```diff
@@ -37,15 +37,15 @@ add_requires() {
 # starts the unit's dependencies. We want to start networkd only on first
 # boot.
 if $(cmdline_bool coreos.first_boot 0); then
-#   add_requires ignition-disks.service
+    add_requires ignition-disks.service
```

Comment:
We were having issues with the ignition-disks stage, why change that in this PR?   Is it required to mount `/boot` or so?

Reply:
Not it is not required. I was under the assumption that all the commented out lines would eventually be uncommented and integrated in. I guess this is not the case. I will remove this.